spec/src/main/asciidoc/microprofile-opentracing.asciidoc

```diff
@@ -192,12 +192,12 @@ The configuration does not disable any outbound request made from the disabled
 server endpoint.
 
 The skip pattern is specified as a string with key `mp.opentracing.server.skip-pattern` which has to be
-compliant with `java.util.regex.Pattern`. An example skip pattern might be `mp.opentracing.server.skip-pattern=/foo|/bar.*`
+compliant with `java.util.regex.Pattern`. An example skip pattern might be `mp.opentracing.server.skip-pattern=/foo|/bar.$$*$$`
```

Comment:
@pilhuhn is this the correct way to escape `*`?

Reply:
I forgot this is an asciidoc rather than md.